src/components/events-list/warning-state.js

```diff
@@ -6,19 +6,11 @@ import eventModel from '../../models/event';
 import { isEventObsolete, isEventResolved } from '../../utils/event-model';
 
 const WarningResolved = () => {
-  return (
-    <React.Fragment>
-      <StatusBadge type="success">Resolved</StatusBadge>
-    </React.Fragment>
-  );
+  return <StatusBadge type="success">Resolved</StatusBadge>;
 };
 
 const WarningObsolete = () => {
-  return (
-    <React.Fragment>
-      <StatusBadge type="warning">Obsolete</StatusBadge>
-    </React.Fragment>
-  );
+  return <StatusBadge type="warning">Obsolete</StatusBadge>;
```

Comment:
Could be deleted if we use `StatusBadge` directly in `event-summary`

Reply:
This file was deleted in the refactor